little-books/src/components/AddBlog/AddBlog.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { FormEvent, useEffect, useState } from "react";
+import { useAppSelector, useAppDispatch } from "../../hooks";
+import { addNewBlog, getFilteredBlogs } from "../../redux/blogs/blogSlice";
+import { BlogType } from "../../types/blog.type";
+import "./AddBlog.scss";
+import { APP_CONSTANTS } from "../../constants/constants";
+import Button from "../Button/Button";
+
+interface ModalProps {
+	handleCloseModal: () => void;
+}
+
+interface Errors {
+	title?: string;
+	details?: string;
+}
```

Comment:
Kindly Explore the difference between interface & type in typescript

Reply:
resolved